Sources/SwiftRuntimeHooks/shims.c

```diff
@@ -0,0 +1,89 @@
+#include <stdlib.h>
```

Comment:
Needed?
I think we only need `<stddef.h>` for `NULL` and `<stdint.h>` for `uint32_t`.